test/twine_test.coffee

```diff
@@ -209,6 +209,16 @@ suite "Twine", ->
       assert.equal node.getAttribute('g'), '0'
       assert.equal node.getAttribute('h'), 'false'
 
+  suite "data-bind-indeterminate attribute", ->
+    test "should set the indeterminate attribute", ->
+      testView = "<input type=\"checkbox\" data-bind-indeterminate=\"key\">"
+      node = setupView(testView, context = key: true)
```

Comment:
Since it's a boolean attribute it should work with values other than strictly true and false. Can you test with other truthy/falsy values, particularly null and undefined?